specification/common-types/resource-management/v2/types.json

```diff
@@ -496,7 +496,7 @@
         "lastModifiedAt": {
           "type": "string",
           "format": "date-time",
-          "description": "The type of identity that last modified the resource."
+          "description": "The timestamp when the resource was last modified (UTC)."
```

Comment:
Hi @jennyhunter-msft, Can you also fix the same error  in v1/type.json?

Reply:
Done